### PR TITLE
Update  `shm_size` details for apcu.md

### PR DESCRIPTION
**Pull request by codyfishman**
Updating old language about increasing `shm_size` that is no longer accurate.

### CODE REVIEW

File: source/content/apcu.md
Comment (stevector) [inline source/content/apcu.md:25]:
```suggestion
```

Below we are just telling people what the values are.

**Comment by rachelwhitton** [comment]
Holding for SME review, todo: update codeowners file before merging

**Review by stevector — COMMENTED** [review]
@rachelwhitton can you look at these copy edits?

**Review by scottbuscemi — APPROVED** [review]

**Pull request merged**
